default_build_settings.cmake

```diff
@@ -130,6 +130,8 @@ function(vespa_use_default_build_settings)
     message("-- CMAKE_SYSTEM_PROCESSOR = ${CMAKE_SYSTEM_PROCESSOR}")
     if(CMAKE_SYSTEM_PROCESSOR STREQUAL "x86_64")
       if(APPLE AND (("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang") OR ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "AppleClang")))
+      elseif("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
+        set(DEFAULT_VESPA_CPU_ARCH_FLAGS "-march=westmere -mtune=haswell")
```

Comment:
Foreslår at du sette skylake som default der. Finnes i praksis ikke noe eldre for apple.
Also add a comment why.

Reply:
Added comment.